css/index.css

```
@@ -17,3 +17,32 @@
 
 
 /* Footer */
+footer
```

Better convention is:
```suggestion
footer {
    ...styles
}
```
Fix for other places in this file